src/main/kotlin/org/sd_jwt/SdJwt.kt

```diff
@@ -366,6 +496,38 @@ inline fun <reified T> createPresentation(
     return presentation
 }
 
+/**
+ * This method takes an SD-JWT and its disclosures and
+ * creates a presentation that discloses only the desired claims.
+ *
+ * @param credential    A string containing the SD-JWT and its disclosures concatenated by a period character
+ * @param claims        A JSONObject of the Credential containing all claims, used for validation
```

Comment:
We only need the structure here without the claims right? In this case the description should be more specific.